src/libstd/panicking.rs

```diff
@@ -149,6 +149,18 @@ impl<'a> PanicInfo<'a> {
         self.payload
     }
 
+    /// Returns the payload as a slice if it is a `&'static str` or `String`.
```

Comment:
Sure, what phrasing would you prefer? (What types do you expect to add? I don’t think I’ve ever seen code panicking with anything else)